.github/workflows/on-push.yml

```diff
@@ -33,9 +33,9 @@ jobs:
     - name: Run Pylint
       run: |
         ./ci docker run qctrl/python-build:3.7 poetry run pylint_runner || true
```

Comment:
I think you should be able to remove the `|| true` from here once all the linting issues are solved